CONTRIBUTING.md

```diff
@@ -66,6 +67,31 @@ as shown below:
 
 `go test ./<directory/path>`
 
+## Prometheus configuration
```

Comment:
I think this requires some context. When do people are going to need this?
I'm not convinced this is something we want to maintain as part of our documentation.
The reason being that it's not often required for doing work on this repo, and I expect it to go stale before it will be of use to someone.